src/components/Checkout/KivaCardRedemption.vue

```diff
@@ -22,9 +22,9 @@
 				v-show="open"
 				class="accordion-info row small-12 columns"
 			>
-				<div class="small-9 small-offset-3 large-10 large-offset-2 columns">
+				<div class="small-12 large-10 large-offset-2 columns">
```

Comment:
Could you please remove the `row` class and all `small-, large-, etc` classes which are rooted in Foundation. You'll need to replace them with the tailwind implementation of flex box. to update the layout of this section of the page.

Reply:
Oh I see, I didn't know that, but sure! will do 👍

Reply:
A new ticket will be opened to change those classes as it's has been discussed